python-lib/dku_config/stl_config.py

```diff
@@ -0,0 +1,163 @@
+from dku_config.decomposition_config import DecompositionConfig
+from dku_config.utils import are_keys_in, is_positive_int
+
+
+class STLConfig(DecompositionConfig):
+    """Mapping structure containing the parameters of the STL decomposition
+
+    Attributes:
+        config(dict): Dict storing the DSSParameters
+        minimum_period: Minimum period required by STL
+    """
+    def __init__(self):
+        super().__init__()
+        self.minimum_period = 2
+
+    def _load_advanced_parameters(self, config):
+        seasonal = config.get("seasonal_stl", 7)
```

Comment:
Because there is a "minI" parameter in the recipe.json, if a user selects a value lower than 7 in the UI, the "seasonal_stl" parameter won't exist and the recipe will work with seasonal_stl=7 instead of failing with a specific error. Is this the expected behavior ?

Reply:
nope... it is so annoying that the value is not kept when it is invalid, I guess the best option is to make it mandatory

Reply:
I agree